sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/quickpulse/model/QuickPulseDocument.java

```diff
@@ -63,4 +57,62 @@ public Map<String, String> getProperties() {
     public void setProperties(Map<String, String> properties) {
         this.properties = properties;
     }
+
+    @Override
+    public JsonWriter toJson(JsonWriter jsonWriter) throws IOException {
+        return jsonWriter.writeStartObject()
+            .writeStringField("__type", type)
+            .writeStringField("DocumentType", documentType)
+            .writeStringField("Version", version)
+            .writeStringField("OperationId", operationId)
+            .writeMapField("Properties", properties, JsonWriter::writeString, true)
```

Comment:
you can delete true. 
this requires us to modify the auto-generated code. to fix that we fixed our agent not to put empty value in the map for our telemetry item exporter pipelines.